Bottom: 29ad88f5f8a8118889256cc8f2e07db96b0c4bb2
Top:    8b8244150b5d8d574a685bd2d96308e521a18d4e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-17 03:08:43 +0100

Refresh of add-support-for-depth-to-trian

---

diff --git a/src/hid/common/hidgl.h b/src/hid/common/hidgl.h
index fc1f37a..aa2985e 100644
--- a/src/hid/common/hidgl.h
+++ b/src/hid/common/hidgl.h
@@ -62,7 +62,7 @@ hidgl_add_triangle (triangle_buffer *buffer,
                     GLfloat x3, GLfloat y3)
 {
   hidgl_add_triangle_3D (buffer, x1, y1, global_depth,
-                                 x2, y2, gloabl_depth,
+                                 x2, y2, global_depth,
                                  x3, y3, global_depth);
 }
